tests/questionTypes.js

```diff
@@ -188,6 +188,7 @@ define([
                             done();
                         }
                     },
+                    features: { case_micro_image: true }
```

Comment:
Why is it needed to add `case_micro_image` here? (I don't have any context on how vellum works, but seeing this here and no other existing features makes me wonder whether I'm missing something)

Reply:
The reason for adding `case_micro_image` here is because we hide the new mug behind a feature flag. We check whether a domain has the necessary flag enabled by looking at the list of features sent by [this](https://github.com/dimagi/commcare-hq/blob/a331be04b86657e62c4ce19cfd4b21571eafc81b/corehq/apps/app_manager/views/formdesigner.py#L311) HQ function. Since the HQ function doesn't get called when running tests, we need to mock having the feature flag enabled for the tests.

Reply:
Ah, I see. Thanks for clarifying.

Reply:
So we need to set it true for all tests by default?

Reply:
No, this only needs to be doneeor tests that include the new `MicroImage` mug type. Currently, only the tests I have added use this new mug type.